src/pages/Dashboard.vue

```diff
@@ -320,6 +336,10 @@ export default {
   },
   computed: {
     ...mapState("sync", ["pending"]),
+    // does plio has any survey question
+    isSurvey() {
```

Comment:
wrong name - use a better one that actually conveys what this variable should do.